contribs/gmf/src/directives/partials/layertree.html

```diff
@@ -1,10 +1,19 @@
 <span ng-if="::!layertreeCtrl.isRoot"
     ng-class="['depth-' + layertreeCtrl.depth, gmfLayertreeCtrl.getResolutionStyle(layertreeCtrl.node), gmfLayertreeCtrl.getNoSourceStyle(layertreeCtrl)]">
   <a href ng-click="::gmfLayertreeCtrl.toggleActive(layertreeCtrl)">
+    <span class="layerIcon">
```

Comment:
I use this class to align the line with others (this `span` element has always a width of 20px).